css/index.css

```diff
@@ -1,3 +1,520 @@
+	margin-right: 20px;
+	width: 45%;
+}
+
+.info-content .about {
+	padding-top: 10px;
+	margin-left: 20px;
+	width: 45%;
+}
+
+.middle-img {
+	margin: 35px 0;
+	width: 100%;
+}
+
+.info-content .ser-pro-vis {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .services {
+	margin-right: 10px;
+	width: 30%;
+	padding-bottom: 25px;
+}
+
+.info-content .product {
+	margin: 0 5px;
+	width: 30%;
+}
+
+.info-content .vision {
+	margin-left: 10px;
+	width: 30%;
+}	
+
+footer {
+	text-align: center;
+	margin-top: 100px;
+}
+
+/* And so begins the tablet break */
+
+@media (max-width: 800px){
+
+    .container {
+        width: 800px;
+    }
+
+    header {
+        height: 130px;
+        width: 800px;
+        position: fixed;
+        top: 0;background-image: none;
+        z-index: 10;
+        display: flex;
+        flex-direction: column;
+    
+    }
+
+    header nav {
+        height: 60px;
+        width: 800px;
+        display: flex;
+        position: relative;
+        line-height: 25px;
+        flex-direction: row;
+        
+    }
+    
+    header nav a {
+        margin: 95px 55px 0 28px;
+        text-decoration: none;
+        justify-content: flex-start;
+        color: rgb(159, 159, 180);
+        line-height: 36px;
+        font-size: 16px;
+        font-weight: bold;
+    }
+
+    
+    header .logo {
+        margin-bottom: 65px;
+        height: 36px;
+        width: auto;
+        align-self: center;
+    }
+    
+    .main-content .snippy {
+        display: none;
+    }
+    
+    .main-content .stack-text {
+        position: absolute;
+        margin-top: 130px;
+        margin-left: 160px;
+        height: 310px;
+        text-align: center;
+        line-height: 73px;
+        font-size: 73px;
+    }
+
+    .main-content .start {
+        position: absolute;
+        border: 1px solid gray;
+        left: 315px;
+        bottom: 67px;
+        padding: 8px 40px 3px 43px;
+        height: 33px;
+        font-size: 17px;
+        text-align: center;
+        font-weight: bold;
+	}
+	
+	.toppy {
+		border-bottom: 2px solid black;
+		margin: 7px 21px 0px 21px;
+	}
+
+	h2 {
+		font-size: 16px;
+	}
+	
+	.info-content .feature-about {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	
+	.info-content .feature {
+		padding-top: 37px;
+		margin-right: 10px;
+		margin-left: 30px;
+		width: 350px;
+		line-height: 18px;
+		font-size: 16px;
+	}
+	
+	.info-content .about {
+		padding-top: 37px;
+		width: 370px;
+		line-height: 18px;
+	}
+	
+	.middle-img {
+		margin: 30px 21px 30px 11px;
+		height: 156px;
+		width: 760px;
+	}
+	
+	.info-content .ser-pro-vis {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	
+	.info-content .ser-pro-vis .services {
+		margin-right: 0px;
+		margin-left: 30px;
+		padding-right: 20px;
+		height: 200px;
+		width: 235px;
+		padding-bottom: 0px;
+		line-height: 18px;
+	}
+	
+	.info-content .product {
+		margin: 0 0px 0px 15px;
```

Comment:
I love the shorthand but I would have just went with a margin-left 15px unless you were just trying to reset the properties from a previous selection. You can also go as far as placing a margin: 0 with a margin-left: 15px below it to let css cascade and take care of clearing out all the others.